card/lib/card/query/attributes.rb

```diff
@@ -27,7 +26,7 @@ def right val
       def editor_of val
         act_join = Join.new(
           from: self,
-          to: ['card_acts', "a#{table_id force=true}", 'actor_id' ]
+          to: ['card_acts', "a#{table_id force=true}", 'actor_id']
```

Comment:
Useless assignment to variable - `force`.<br>Surrounding space missing for operator '='.